Replaced some functions(should make it more easy to use for people).

BadMaliciousPyScripts opened the pull request:
Replaced queuing with auto accepting, tft lobby creation, play again and changed the surrendering function.
I am mainly using LCU in my changes.

Detergent13 commented:
Hey! Before I start, I just want to thank you for your contribution- I appreciate it, great to see someone adding to the repository.

I didn't even think of using the LCU to do this, but definitely a more elegant solution than using computer vision like I was. 

That being said, I did find a slight issue as I tested the changes- not sure why it's not working for me and assumedly is for you, but the TFT window doesn't seem to accept keyboard input from pyautogui for some reason, meaning the new way of surrendering (typing /ff in chat) doesn't work. I tried to get around this back when I wrote the bot initially by using a wrapper method, `click_key()`, but no luck. Might have something to do with fullscreen vs. windowed borderless vs. windowed?

If we can find a fix for that small issue, I'd be all good to merge this PR. Also would be good to get rid of the image recognition for rerolling and just use the 'D' key to reroll instead, if we can figure out how to make keyboard input work universally. 

Also just fixed a couple of super minor grammar mistakes and changed the auto_gui class to a more fitting name- sorry to nitpick!

Detergent13 commented on tft.py:
```diff
@@ -1,168 +1,285 @@
+        request = requests.post(url, headers=headers, data=data, verify=False)
+        return request.status_code
+
+    def create_tutorial_lobby_tft(lcu_data):
```
Just curious, what's this method for? Just in case somebody wants to bot tutorial games instead?

BadMaliciousPyScripts replied:
Ok I will look into the league window not accepting keyboard input issue as soon as i have free time. ^^

BadMaliciousPyScripts replied:
> Just curious, what's this method for? Just in case somebody wants to bot tutorial games instead?

For testing purposes only
So I dont have to go into a actual game to test things

BadMaliciousPyScripts commented:
> Also just fixed a couple of super minor grammar mistakes and changed the auto_gui class to a more fitting name- sorry to nitpick!

Most of the fixed grammar mistakes just came from the way I write so don't worry and the unvalid came from me hitting the wrong button cause both u and i are aligned next to each other on my keyboard and that u changed the auto_gui class to wrapper is fine too its your main code after all so don't be sorry ^^

In my next commit I would like to have already the picking up of orbs implemented. Is this in any way a problem for you or are you already working on it? If not I would be happy to do it.

Detergent13 commented:
For anybody watching the PR- should be ready to merge once we get finishing touches done with (really just finding an LCU endpoint for the missions 'OK' button and 'skip waiting for stats' button). Orb collection is coming when it's merged and possibly items some time in the future!

BadMaliciousPyScripts commented:
It is probably easier to open a new pr because u can't Auto merge cause of the readme changes. So I will open a new one. Seems kinda messy but what ever.
